client/src/components/Toasts/Toasts.scss

```diff
@@ -0,0 +1,157 @@
+.toasts {
+  position: fixed;
+  top: 60px;
+  right: 20px;
+  width: 250px;
+  z-index: 9999;
+
+  &__notice-wrap {
+    position: absolute;
+  }
+}
+
+
+@keyframes toastFadeIn {
+  0% { opacity: 0; }
+  100% { opacity: 1; }
+}
+
+@keyframes toastFadeOut {
+  0% {
+    opacity: 1;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  50% {
+    opacity: 0;
+    max-height: 250px;
+    margin-bottom: map_get($spacers, sm);
+  }
+
+  100% {
+    opacity: 0;
+    max-height: 0;
+    margin-bottom: 0;
+  }
+}
+
+
+.toast {
+  margin: 0 0 map_get($spacers, sm) 0;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: inherit;
```

Comment:
`font-family` really doesn't seem to do anything here. Removing `font-size` reduces the size of the content, but I think that'd make more sense to be directly set in `.toast__content`.

Reply:
I've looked up the base bootstrap style https://github.com/twbs/bootstrap/blob/2bceb3e79d848a101e7aa5fc4b2020e681b1dfc3/scss/_toasts.scss

I'll override the bootstrap variables instead.